lib/utilities/RelayMD5Sum.js

```diff
@@ -0,0 +1,48 @@
+import crypto from 'crypto';
+import { Transform } from 'stream';
+
+/**
+ * This class is designed to compute the md5 hash on multiple streams in relay
+ * race style.
+ */
+export default class RelayMD5Sum extends Transform {
+
+    /**
+     * @constructor
+     * @param {Hash} starterHash - hash from prior stream (if any) to be updated
+     * with new stream here
+     * @param {function} done Callback - This callback is called when
+     * the hash computation is finished, this function need to be synchronous
+     * and being call before the end of the stream object
+     */
+    constructor(starterHash, done) {
```

Comment:
starterHash is the only difference with the class MD5Sum, can we just adapt this one ?